pdns/dnsdist-protobuf.cc

```diff
@@ -33,7 +34,7 @@ static void protobufFillMessage(PBDNSMessage& message, const DNSQuestion& dq)
   message.set_inbytes(dq.len);
 
   struct timespec ts;
-  clock_gettime(CLOCK_REALTIME, &ts);
+  gettime(&ts);
```

Comment:
Yes, one of the reasons this is not ready for merge, I'm flattening the different timescales. A bool sounds good; I was thinking about an enum (mirroring the original `clock_gettime` definition) but a bool is probably fine.